### DISCUSSION TITLE
I want a `vertical-align` property that aligns with the text middle

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#206] [Wants]
Icons and other glyph-like symbols appear most consistent with adjacent text when they are aligned with that text&#39;s typographic middle (half its cap height). There is currently no `vertical-align` property that easily accomplishes this: Many projects instead use `vertical-align: middle` and then push or pull the icon up or down a pixel or two using `top`, `translate` or `margin` to account for the difference. It would be nice to have an explicit `vertical-align: text-middle` property.

Here’s [an example of what that could look like](https://cloudfour.com/thinks/some-imaginary-css/#align-to-typeface-median).

<hr>

https://webwewant.fyi/wants/113/

**Comment by miragecraft** [comment]
The upcoming leading-trim CSS property should fix this, right?
